vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -1,53 +1,48 @@
 # Entando Identity Management -- Keycloak
 
-Entando Identity Management is powered by Keycloak. The Keycloak instance used for your Entando apps
-can be [externally installed](../../tutorials/devops/external-id-management) or you can use a dedicated instance on a per application basis. The sections below details the architecture and documentation required to customize your Keycloak instance.
+Entando Identity Management authenticates Entando Applications with Keycloak. The Keycloak instance can be either [externally installed](../../tutorials/devops/external-id-management) or dedicated on a per-application basis. The architecture and requirements to customize your Keycloak instance are described below.
 
 ## Logging into your Keycloak Instance
 
-In an Entando deployment Keycloak is protected by a Secret deployed in your Kubernetes instance. To get the default admin credentials you can query Kubernetes for the secret with this command:
+In an Entando deployment, Keycloak is protected by a Secret in your Kubernetes instance. You can query Kubernetes for the Secret's default admin credentials, substituting your environment's namespace and Secret name:
 
 ```
 kubectl get secret quickstart-kc-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
 ```
 
-Update the namespace and secret name to match your environment. If you're not sure of the secret name you can run
+ To find the Secret name, run
 ```
 kubectl get secrets -n <namespace>
 ```
-And search for the secret that ends in `kc-admin-secret`
+and search for the Secret that ends in `kc-admin-secret`.
 
 ## Authentication
-In Entando 6 all authentication goes through Keycloak. This ensures that a micro frontend can call a microservice with a token that is available on the client.
+Beginning with Entando 6, all authentication is powered by Keycloak. This ensures that a micro frontend calls a microservice with a token that is available on the client.
```

Comment:
"can call" is actually better since there's no requirement that the MFE or MS requires a Keycloak token. It's available but not required.